<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParagraphBulletsNumbers</class>
 <widget class="QWidget" name="ParagraphBulletsNumbers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>386</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="listTypes"/>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="listPropertiesPane">
     <layout class="QGridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="3">
       <layout class="QGridLayout">
        <item row="0" column="3">
         <widget class="QLineEdit" name="suffix"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Start at:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>startValue</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="ListsSpinBox" name="startValue">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Alignment:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>alignment</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Prefix:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>prefix</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="prefix"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Suffix:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>suffix</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QComboBox" name="labelFollowedBy"/>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="alignment"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Depth:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>depth</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="depth">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Display Levels:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>levels</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QSpinBox" name="levels">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Bullet/Number followed by</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QDoubleSpinBox" name="doubleSpinBox">
          <property name="suffix">
           <string> cm</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>at</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
          <property name="suffix">
           <string> cm</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Aligned at</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
          <property name="suffix">
           <string> cm</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Indented at</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Custom character:</string>
        </property>
        <property name="buddy">
         <cstring>customCharacter</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="customCharacter">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>251</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="13" column="0" colspan="3">
       <widget class="QCheckBox" name="restartNumbering">
        <property name="text">
         <string>Restart numbering at this paragraph</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="3">
       <widget class="QCheckBox" name="letterSynchronization">
        <property name="text">
         <string>Letter Synchronization</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <layout class="QGridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="widthLabel">
          <property name="text">
           <string>Width</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="insertImageLabel">
          <property name="text">
           <string>Insert Image</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="imageWidth"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="heightLabel">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QDoubleSpinBox" name="imageHeight"/>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="insertImage">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="15" column="0" colspan="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ListsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>dialogs/ListsSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
